packages/js/test/plugins/auctionHouseModule/withdrawFromBuyerAccount.test.ts

```diff
@@ -0,0 +1,70 @@
+import test, { Test } from 'tape';
+import { sol } from '@/types';
+import { metaplex, killStuckProcess, assertThrows } from '../../helpers';
+import { createAuctionHouse } from './helpers';
+import { findAuctionHouseBuyerEscrowPda } from '@/plugins';
+import { Keypair } from '@solana/web3.js';
+
+killStuckProcess();
+
+test('[auctionHouseModule] withdraw from buyer account on an Auction House', async (t: Test) => {
+  // Given we have an Auction House.
+  const mx = await metaplex();
+
+  const { auctionHouse } = await createAuctionHouse(mx);
+
+  // Deposit 1 SOL.
+  await mx
+    .auctionHouse()
+    .depositToBuyerAccount({
+      auctionHouse,
+      amount: sol(1),
+    })
+    .run();
+
+  // And withdraw 1 SOL.
```

Comment:
```suggestion
  // When we withdraw 1 SOL from the buyer's escrow account.
```

Tests must be
_Given
And...
When
Then
And..._